docs/README.md

```diff
@@ -215,6 +216,7 @@ for more information about installing and using the source code.
 | [Folder create directive](content-services/folder-create.directive.md) | Allows folders to be created. | [Source](../lib/content-services/folder-directive/folder-create.directive.ts) |
 | [Folder edit directive](content-services/folder-edit.directive.md) | Allows folders to be edited. | [Source](../lib/content-services/folder-directive/folder-edit.directive.ts) |
 | [File draggable directive](core/file-draggable.directive.md) | Provide drag-and-drop features for an element such as a `div`. | [Source](../lib/content-services/upload/directives/file-draggable.directive.ts) |
+| _Inherited button directive_ | _Not currently documented_ | [Source](../lib/content-services/permission-manager/components/inherited-button.directive.ts) |
```

Comment:
I think we have it https://github.com/Alfresco/alfresco-ng2-components/blob/development/docs/content-services/inherit-permission-directive.md